src/io/files.js

```diff
@@ -177,6 +177,8 @@ p5.prototype.loadFont = function (path, onSuccess, onError) {
  *                                    there is an error, response is passed
  *                                    in as first argument
  * @return {Object|Array}             JSON data
+ * @return {array}                    Array
+ * @return {integer}                   Integer
  * @example
  *
```

Comment:
I added error handling so if `resp` received a `type: array` or `type: integer` it would be parsed as a string... I realized this was a little naive since array's have a bit more to them than that. Would adding a `map('text/array')` functionality to `loadJSON` to store the `array` this seems like a good solution. @lmccart @limzykenneth curious to hear your feedback on this :)